packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -127,17 +127,25 @@ const prepareXAxis = (
   width,
   format,
   orientation = AXES_ORIENTATION.X.BOTTOM,
+  lowerLimit = null,
+  upperLimit = null,
```

Comment:
Why are these being defaulted to null? This is effectively evaluating the value against 0 which may not be correct.